.github/workflows/ci_cd_release.yml

```
@@ -132,9 +152,11 @@ jobs:
 
     - name: "Install Git and clone project"
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        persist-credentials: false
 
     - name: "Set up Python"
-      uses: ansys/actions/_setup-python@main
+      uses: ansys/actions/_setup-python@feat/check-actions-security-action
```

```suggestion
      uses: ansys/actions/_setup-python@main
```
Needs to be reverted before merging